src/routing_table/network_tests.rs

```diff
@@ -115,34 +130,44 @@ impl Network {
             }
         }
 
+        let mut iteration = 1;
         while !merge_own_info.is_empty() {
-            let mut merge_other_info = Vec::new();
+            trace!("+++ Merge own info: iteration #{}", iteration);
```

Comment:
Do we still need the `trace!` statements and `iteration` counter?  They're not really useful unless we also initialise logging in the test, right?